docs/examples/sequence_processing/optical_flow_example.ipynb

```diff
@@ -24,8 +24,8 @@
     "import os.path\n",
```

Comment:
<p>I'm not sure what is the convention we want to follow, but I think that using absolute url to the docs (like https://docs.nvidia.com/deeplearning/dali/user-guide/docs/supported_ops.html#nvidia.dali.fn.readers.video</p><p>) will always land you in current release docs rather than the one you are looking at.</p><p>For example, clicking this in your local build takes you online. And going through this page for the docs for some old release and clicking it will take you to current one.</p><p> </p><p>Same goes for other links added in this PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>